.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Debug Asterinas(debug)",
+            "type": "lldb", // install CodeLLDB extension first
```

Comment:
Interesting. So it seems that the VS code supports JSON format with comment extension.